### DISCUSSION TITLE
docket-entries API endpoint vs docket_absolute_url from archive search

### DISCUSSION THREAD

**Discussion by anthonysalvato3** [freelawproject/courtlistener#2845] [Q&A] [answered]
I understand using the docket-entries endpoint requires special permission. My use case is I'm doing a RECAP archive search with specific criteria, then I'm looking up the resulting docket numbers to check the filings. I noticed I can just use the docket_absolute_url from the response of the archive search to do that. Would there be an advantage to instead requesting permission for the docket-entries API?

**Comment by mlissner** [comment, marked as answer]
I think it depends on if you want data or HTML. If you use the absolute_url field, you'll get our rendering of the data. If you want *data*, you'll want to get it as JSON and use that. 

Maybe I'm still not understanding correctly?

**Reply by anthonysalvato3** [reply]
Makes sense. To email for permission, do I use the contact at the bottom of the readme, or is there a different contact specific to CourtListener?

**Reply by mlissner** [reply]
I'll get back to you in a few. :)

Thanks for moving this over here even though I see it's a bit circuitous now...
